.github/file-filters.yml

```diff
@@ -29,13 +29,16 @@ frontend_components_modified_lintable:
 backend_lintable: &backend_lintable
   - '**/*.py'
 
+backend_dependencies: &backend_dependencies
+  - 'requirements-*.txt'
```

Comment:
This is what we'll use to determine if a dispatch to getsentry is required.

Reply:
You should probably add that as a comment.